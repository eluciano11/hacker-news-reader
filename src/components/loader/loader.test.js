import React from "react";
import { render, cleanup } from "@testing-library/react";

import Loader from "./Loader";

afterEach(cleanup);

describe("Loader Component Tests", () => {
  it("Should render loader", () => {
    const { asFragment } = render(<Loader />);
    expect(asFragment()).toMatchSnapshot();
  });
});
