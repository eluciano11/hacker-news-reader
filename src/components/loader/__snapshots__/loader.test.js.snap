// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component Tests Should render loader 1`] = `
<DocumentFragment>
  <div
    class="loader small"
  />
</DocumentFragment>
`;
